### DISCUSSION TITLE
how to rotate briding?

### DISCUSSION THREAD

**Discussion by tome9111991** [supermerill/SuperSlicer#1997] [Q&A] [answered]
hi
I have a part here where I want to rotate the briding by 90 °
acctual looks: (light blue)
![Screenshot 2021-12-03 163654](https://user-images.githubusercontent.com/57866234/144630383-326488f5-addf-466a-820f-076c82d92385.png)

the briding will start in the air.
when i print the front part sags.

my idea to rotate the briding about 90°
to start the briding above a solild.

[rotate_bridge.zip](https://github.com/supermerill/SuperSlicer/files/7650666/rotate_bridge.zip)

**Comment by supermerill** [comment, marked as answer]
infill -> advanced -> angel -> bridging

or just search (magnify glass) for "angle" (it's the second result)

**Comment by tome9111991** [comment]
I tried that.
did not get the expected result.
looks:
![90grad](https://user-images.githubusercontent.com/57866234/144635463-23426fbd-1869-46ba-9a20-766ef90d0a10.png)

i use latest release version

**Reply by supermerill** [reply]
![image](https://user-images.githubusercontent.com/6536403/144644213-b41f8506-731f-41f8-9713-b3aa3913a3c8.png)

Your picture is that way because of 'Extra perimeters' 'On Overhangs'

**Reply by tome9111991** [reply]
Thanks  'Extra perimeters' 'On Overhangs' was the issue
